packages/thirdweb/src/exports/indexer.ts

```diff
@@ -0,0 +1,6 @@
+export { getIndexerClient } from "../indexer/indexer";
+export { getBlocks } from "../indexer/endpoints/getBlocks";
+export { getEvents } from "../indexer/endpoints/getEvents";
+export { getLatestBlock } from "../indexer/endpoints/getLatestBlock";
+export { getReceipts } from "../indexer/endpoints/getReceipts";
+export { getNFTs } from "../indexer/endpoints/getNFTs";
```

Comment:
I wonder if these should be explicit exports or if we should instead use them "under the hood" in less indexer-specific places. I guess there is always a likely requirement to also *explicitly* be able to ask the indexer for data

Reply:
We can both export and use them under the hood